content/en/agent/proxy.md

```diff
@@ -134,20 +134,114 @@ proxy_password: my_password
 
 Do not forget to [restart the Agent][1] for the new settings to take effect.
 
+## Squid
 
+[Squid][2] is a caching proxy for the web supporting HTTP, HTTPS, FTP, and more. It runs on most available operating systems, including Windows, and is licensed under the GNU GPL. Squid is easy to configure and widely used for proxying traffic in a variety of different situations. 
+
+This is the best option if you do not have a web proxy readily available in your network and you wish to proxy a large number of Agents. 
```

Comment:
@Kaderinho Those are fair concerns, I see where you're coming from when it comes to putting this under the Web Proxy section; but I think that maybe it may make sense to have a section called "Other Web Proxies," and then have a dedicated section for Squid (or have two sections under Web Proxy: "Squid" and "Other Web Proxies"). 

Customers starting from scratch typically jump straight to the Nginx/HAProxy section so it would still be useful to link them directly. 

>I don't think that Squid is the best option here especially because one of its main purpose is to cache web pages and therefore improving performances of a web server while our backend (AFAIK) will not send any static file to the Agent (so there's basically nothing to cache here).

You could make the same argument with HAProxy being traditionally used as a load balancer. You don't necessarily need to use the caching features of Squid in this case. 

>might actually need to benchmark Squid vs HAProxy with the Agent to have a good understanding of its impact

What is involved with doing this benchmarking? Many folks in the field are already recommending Squid to customers and have already implemented this as the proxying solution for at least the past couple of years.